desktop/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "WordPressDesktop",
-	"version": "8.0.0",
+	"version": "8.1.0",
```

Comment:
By convention we typically save the version bump for an actual release PR. The release itself should be isolated from any other code-changes.

Reply:
Interesting, I thought a major bugfix like this would require a version bump of some sort.

Reply:
Reverted in 3e0a8ece862d8eb2f51604ff715eea2a7515c195.